### PR TITLE
Allow to not render the logging sections in zope.conf

### PR DESCRIPTION
**Pull request by frapell**

### CODE REVIEW

**Comment by jensens** [comment]
i dont get it exactly. with no section does it mean no logging at all? a little more verbose explanation would be great!

**Comment by frapell** [comment]
@jensens that is exactly the case. using a value of 'disable' for either the event-log or the z2-log will cause the proper sections to be omitted, disabling the logging.

There are some use cases where you want them disabled, and right now the recipe doesn't allow a way to do it, since it will always include an <eventlog> and a <logger access> sections

File: README.rst [hunk 1]
```diff
@@ -221,6 +221,7 @@ log levels or configure `mailinglogger`.
 
 event-log
   The filename of the event log. Defaults to ${buildout:directory}/var/log/${partname}.log
+  Setting this value to 'disable' will make the <eventlog> section to be omitted
```
Comment (jensens) [inline README.rst]: so may you add here: ".. omitted, resulting in no logging at all."

File: README.rst [hunk 2]
```diff
@@ -251,6 +252,7 @@ mailinglogger
 
 z2-log
   The filename for the Z2 access log. Defaults to var/log/${partname}-Z2.log.
+  Setting this value to 'disable' will make the <logger access> section to be omitted
```
Comment (jensens) [inline README.rst]: same as above

**Comment by jensens** [comment]
useful indeed! see code comments to really explain it also for people like me ;)

**Comment by frapell** [comment]
@jensens Agree, do you think that wording is ok?

**Comment by jensens** [comment]
at least i understand it ;) i'am not a native speaker, just one of those germans. I would say its perfect.

**Pull request merged**